e2e/framework/test_context.go

```diff
@@ -209,10 +208,11 @@ type TestContextType struct {
 }
 
 type BandwagonConfig struct {
-	Organization string `json:"organization" yaml:"organization" `
-	Username     string `json:"username" yaml:"username" `
-	Password     string `json:"password" yaml:"password" `
-	Email        string `json:"email" yaml:"email" `
+	Organization string `json:"organization" yaml:"organization" env:"ROBO_BANDWAGON_ORGANIZATION"`
+	Username     string `json:"username" yaml:"username" env:"ROBO_BANDWAGON_USERNAME"`
+	Password     string `json:"password" yaml:"password" env:"ROBO_BANDWAGON_PASSWORD"`
+	Email        string `json:"email" yaml:"email" env:"ROBO_BANDWAGON_EMAIL"`
```

Comment:
1. I thought we agreed on eliminating global initializers such as ENV variables. Also you don't need them here. 
2. You don't use JSON tag
3. This config is part of Bandwagon defined as 
```
	// Bandwagon specifies bandwagon creation details
	Bandwagon *BandwagonConfig `json:"bandwagon,omitempty"`
```
- please add `validate` tag to the fields themselves : they either need be filled in in full OR replaced with default one as a whole if it came as `nil` from config file  - don't cherry pick default values as in `func (b *Bandwagon) SubmitForm(config framework.BandwagonConfig)`